python/setup.py

```diff
@@ -223,13 +223,14 @@ def get_packages(self):
         # pyarrow dropped python 3.6 support in 7.0.0.
         # Serialization workaround for pyarrow 7.0.0+ doesn't work for Windows.
         pyarrow_dep = "pyarrow >= 6.0.1, < 7.0.0"
+
+    data_extras = [numpy_dep, pandas_dep, pyarrow_dep, "fsspec"]
+    # Need pickle5 package to override default pickle package for Python < 3.7
+    if sys.version_info < (3, 7):
```

Comment:
Ah this should be `<=` right?
```suggestion
    if sys.version_info <= (3, 7):
```